app/lib/assessment_stats.rb

```diff
@@ -0,0 +1,101 @@
+# RequestLog filtering to generate stats based on https://docs.google.com/spreadsheets/d/1YB_JDQJpq_ZqYRbJ-FH53orLgIcuyBplgrDRl8BgY8M/edit#gid=0
+# :nocov:
```

Comment:
IMHO it would be better to mark this as not covered in spec_helper.rb with a comment. Otherwise it just looks like we're avoiding coverage